src/network/serverpackethandler.cpp

```diff
@@ -1346,20 +1348,37 @@ void Server::handleCommand_RemovedSounds(NetworkPacket* pkt)
 	}
 }
 
+static void sanitize_field(std::string &s)
+{
+	// truncate on NULL
+	auto pos = s.find('\000');
+	if (pos != s.npos)
```

Comment:
```suggestion
	if (pos != std::string::npos)
```

(seems to occur more often in our code, I prefer it because it makes it more obvious that it's `static` and not a member variable)

Reply:
I was just lazy while typing :)